docs/sources/reference/run.md

```diff
@@ -615,6 +616,30 @@ memory nodes 1 and 3.
 This example restricts the processes in the container to only use memory from
 memory nodes 0, 1 and 2.
 
+### Block IO bandwidth (Blkio) constraint
+
+By default, all containers get the same proportion of block IO bandwidth
+(blkio). This proportion is 500. To modify this proportion, change the
+container's blkio weight relative to the weighting of all other running
+containers using the `--blkio-weight` flag.
+
+The `--blkio-weight` flag can set the weighting to a value between 10 to 1000.
+For example, the commands below create two containers with different blkio
+weight:
+
+    $ docker run -ti --name c1 --blkio-weight 300 ubuntu:14.04 /bin/bash
+    $ docker run -ti --name c2 --blkio-weight 600 ubuntu:14.04 /bin/bash
+
+If you do block IO in the two containers at the same time, by, for example:
+
+    $ time dd if=/mnt/zerofile of=test.out bs=1M count=1024 oflag=direct
+
+You'll find that the proportion of time is the same as the proportion of blkio
+weights of the two containers.
+
+> **Note:** The blkio weight setting is only available for direct IO. Buffered IO
+> is not currently supported.
+
```

Comment:
@vbatts Do we still think a note is required for RHEL6 (https://github.com/docker/docker/pull/11802#issuecomment-94032348)? If *really* needed! it could be added here.

On the other hand, if it's a rare case, we *could* ignore it for now and see if people report issues (or beat them to it and create an issue now; an issue on GitHub with a SEO optimised title could work, and we don't have to clutter the docs with yet another "note")